client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/OpenApiConfigValidatorTest.java

```diff
@@ -18,7 +18,11 @@ class OpenApiConfigValidatorTest {
             "quarkus.openapi-generator.codegen.additional-api-type-annotations",
             "quarkus.openapi-generator.codegen.spec.spec_yaml.enable-security-generation",
             "quarkus.openapi-generator.codegen.type-mappings.UUID=String",
-            "quarkus.openapi-generator.codegen.spec.spec_yaml.type-mappings.UUID=String"
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.type-mappings.UUID=String",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-apis=false",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-models=false",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-apis=true",
+            "quarkus.openapi-generator.codegen.spec.spec_yaml.generate-models=true",
```

Comment:
Actually, this is not really testing this new feature. It just ensures the configs don't cause any exceptions.
I'd rather create a new project under https://github.com/quarkiverse/quarkus-openapi-generator/tree/72edb3f873e0478a463b50b7db9e7ad83a9be9a0/client/integration-tests to test specifically if the API and models were created or not, according to both values, `true` and `false`.

Due to these methods not being used, I suspect the feature is not working correctly.

![image](https://github.com/user-attachments/assets/17a0a06d-4f7c-4b71-b27f-62269fef17b0)